contracts/bridge/BridgeTransferCommon.sol

```diff
@@ -0,0 +1,124 @@
+pragma solidity ^0.4.24;
+
+import "../externals/openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+import "./BridgeHandledRequests.sol";
+import "./BridgeFee.sol";
+import "./BridgeOperator.sol";
+
+contract BridgeTransferCommon is BridgeHandledRequests, BridgeFee, BridgeOperator {
```

Comment:
How about `BridgeCommon`? It inherits fee and operator also.

Reply:
This file is for common things of KLAY/ERC20/ERC721 value transfer.
So I named like this.

If we have some common things for Bridge, we can write them on `bridge.sol` like `counterpartBridge`. :)
It is ok?

Reply:
How about `BridgeTransfer`?

Reply:
OK. I will reflect :)